src/index.rs

```diff
@@ -0,0 +1,33 @@
+use std::io;
+use crate::metadata::get_ids;
+
+pub struct Index {
+    pub packets: Vec<Packet>
```

Comment:
this is a struct because down the line we'll want other info aside from just ids?

Reply:
Yeah, so if we follow same structure as in R we'll also store here some data structure for the dependencies between packets. And a `Packet` itself will have info about name, parameters and location